.travis.yml

```diff
@@ -5,108 +5,139 @@
 # @link https://github.com/WordPress/twentynineteen/
 
 # Declare project language and PHP versions to test against.
-# @link http://about.travis-ci.org/docs/user/languages/php/
+# @link https://about.travis-ci.org/docs/user/languages/php/
 language: php
 
 # Newer versions like trusty don't have PHP 5.2 or 5.3
 # https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming
-dist: precise
+dist: trusty
```

Comment:
PHP 5.2 and 5.3 are not available on Trusty, this is why it has remained Precise

Reply:
I have added them under:
```yml
    - php: "5.3"
      env: PHPCS_BRANCH="2.9.0" SHORT_OPEN_TAGS=true PHAR_LINT=1
      dist: precise
    # Current minimum $required_php_version for WordPress: 5.2.4
    - php: "5.2"
      env: PHPCS_BRANCH="2.9.0" SHORT_OPEN_TAGS=true PHAR_LINT=1
      dist: precise
```
So this should be ok. The travis fails on phpcs `wget` tho.

Reply:
Hehe, nice, ignore me 😏

Reply:
Also I noticed that phpcs wasn't running in the previous builds...